HISTORY.md

```diff
@@ -3,6 +3,10 @@
 ### Behavior Changes
 * Attempting to write a merge operand without explicitly configuring `merge_operator` now fails immediately, causing the DB to enter read-only mode. Previously, failure was deferred until the `merge_operator` was needed by a user read or a background operation.
 
+### API Changes
+* `rocksdb_approximate_sizes` in the C API now requires an error pointer (`char** errptr`) for receiving any error.
+* `DB::GetApproximateSizes` now returns a Status object which indicates where the sizes could be correctly obtained.
```

Comment:
"Where" suggests specific Status for each size requested, which is not the case. And 1/3 overloads of DB::GetApproximateSizes already returned Status.

Reply:
Suggest

> All overloads of `DB::GetApproximateSizes` now return Status, so that any failure to obtain the sizes is indicated to the caller.

Reply:
Done.